local-setup/tests/e2e/kc-parity/lifecycle-through-kc.sh

```
@@ -0,0 +1,44 @@
+cu=json.loads('''$CU''')
+print(json.dumps({'RequestInfo':{'apiId':'Rainmaker','msgId':f'{int(time.time())}000|en_IN','authToken':'$CIT','userInfo':cu},
+ 'service':{'tenantId':'$TEN','serviceCode':'${SERVICE:-DamagedRoad}','description':'kc lifecycle probe','source':'web',
+   'address':{'city':'$TEN','locality':{'code':'BOMET_BOMET_CENTRAL_MUTARAKWA','name':'Chesoen'},'geoLocation':{'latitude':-0.7813,'longitude':35.3416}},
+   'citizen':{'name':cu.get('name','Citizen'),'mobileNumber':cu.get('mobileNumber'),'countryCode':cu.get('countryCode'),'emailId':cu.get('emailId'),'type':'CITIZEN','tenantId':'$ROOT','uuid':cu.get('uuid')}},
+ 'workflow':{'action':'APPLY'}}))")")
+SRID=$(echo "$CREATE"|python3 -c "import json,sys;print((json.load(sys.stdin).get('ServiceWrappers') or [{}])[0].get('service',{}).get('serviceRequestId',''))" 2>/dev/null)
+say "APPLY (citizen files)" "${SRID:-FAILED: $(echo "$CREATE"|head -c 110)}"
+[ -z "$SRID" ] && exit 1
+
+step(){ # action token userinfo extra
+  local S=$(curl -s -X POST "$KC/kc/pgr-services/v2/request/_search?tenantId=$TEN&serviceRequestId=$SRID" -H "Content-Type: application/json" -H "Authorization: Bearer $2" -d "{\"RequestInfo\":$(RI "$2" "$3")}" \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)